scripts/exgdas_global_marine_analysis_post.py

```diff
@@ -59,28 +59,33 @@ def list_all_files(dir_in, dir_out, wc='*', fh_list=[]):
 post_file_list.append([os.path.join(anl_dir, 'inc.nc'),
                        os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocninc.nc')])
 
-# Copy of the diagonal of the background error for the cycle
-post_file_list.append([os.path.join(anl_dir, 'static_ens', f'ocn.orig_ens_stddev.incr.{bdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocn.bkgerr_stddev.nc')])
-post_file_list.append([os.path.join(anl_dir, 'static_ens', f'ice.orig_ens_stddev.incr.{bdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ice.bkgerr_stddev.nc')])
-
-# Copy the ice and ocean increments
-post_file_list.append([os.path.join(anl_dir, 'Data', f'ocn.3dvarfgat_pseudo.incr.{mdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ocn.incr.nc')])
-post_file_list.append([os.path.join(anl_dir, 'Data', f'ice.3dvarfgat_pseudo.incr.{mdate}.nc'),
-                       os.path.join(com_ocean_analysis, f'{RUN}.t{cyc}z.ice.incr.nc')])
+domains = ['ocn', 'ice']
+for domain in domains:
+    # Copy of the diagonal of the background error for the cycle
+    post_file_list.append([os.path.join(anl_dir, 'static_ens', f'{domain}.bkgerr_stddev.incr.{bdate}.nc'),
```

Comment:
We should talk tomorrow about using Jinja YAML templates and not having this be scripted. Not important for this PR, but future.

Reply:
ok

Reply:
created issue here: https://github.com/NOAA-EMC/GDASApp/issues/612